automation_tools/satellite6/upgrade/tasks.py

```diff
@@ -353,15 +351,18 @@ def sync_capsule_tools_repos_to_upgrade(admin_password=None):
         print('The Capsule repo URL is not provided '
               'to perform Capsule Upgrade in feature!')
         sys.exit(1)
-    version = distro_info()[1]
+    cv_name, env_name, ak_name = [
+        os.environ.get(env_var)
+        for env_var in ('CAPSULE_CV', 'CAPSULE_ENVIRONMENT', 'CAPSULE_AK')
+    ]
     details = os.environ.get('CAPSULE_SUBSCRIPTION')
     if details is not None:
         cv_name, env_name, ak_name = [
             item.strip() for item in details.split(',')]
-    else:
-        cv_name = 'rhel{0}_cv'.format(version)
-        env_name = 'DEV'
-        ak_name = 'rhel'
+    elif not all(cv_name, env_name, ak_name):
```

Comment:
This will fail

```
>>> all(1, 2, 3)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: all() takes exactly one argument (3 given)
```